SNIPS/snip-6.md

```diff
@@ -85,6 +123,12 @@ Wallets are advised not to sign any data unless they know that it includes the a
 
 ## Backwards Compatibility
 
+As mentioned before the interface ID for an account is always `0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd`** which matches the original account interface. 
```

Comment:
I wouldn't say "always ...." this, for some reason, might be subject to changes later on.
Maybe:
```suggestion
As mentioned before the interface ID for an account is stays `0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd`** which matches the original account interface. 
```